calico/getting-started/kubernetes/managed-public-cloud/aks.md

```diff
@@ -9,15 +9,79 @@ Enable {{site.prodname}} in AKS managed Kubernetes service.
+
+You can also use {{site.prodname}} for networking on AKS in place of the default Azure VPC networking. This allows you to take advantage of the full set of {{site.prodname}} networking features.
 
 ### How to
 
-To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+#### Install AKS with {{site.prodname}} for network policy
 
 The geeky details of what you get:
 {% include geek-details.html details='Policy:Calico,IPAM:Azure,CNI:Azure,Overlay:No,Routing:VPC Native,Datastore:Kubernetes' %}
 
+To enable {{site.prodname}} network policy enforcement, follow these step-by-step instructions: {% include open-new-window.html text='Create an AKS cluster and enable network policy' url='https://docs.microsoft.com/en-us/azure/aks/use-network-policies' %}.
+
+#### Install AKS with {{site.prodname}} networking
+
+The geeky details of what you get:
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:VXLAN,Routing:Calico,Datastore:Kubernetes' %}
+
+1. Create an Azure AKS cluster with no Kubernetes CNI pre-installed. Please refer to [Bring your own CNI with AKS](https://docs.microsoft.com/en-us/azure/aks/use-byo-cni?tabs=azure-cli) for details.
+   ```bash 
+    # Install aks-preview extension
+    az extension add --name aks-preview
+    # Update aks-preview to ensure latest version is installed
+    az extension update --name aks-preview
+
+    # Create a resource group
+    az group create --name my-calico-rg --location westcentralus
+
+    az aks create --resource-group my-calico-rg --name my-calico-cluster --location westcentralus --network-plugin none
+    ```
+
+1. Get credentials to allow you to access the cluster with `kubectl`:
+    ```
+    az aks get-credentials --resource-group my-calico-rg --name my-calico-cluster
+    ```
+
+1. Now that you have a cluster configured, you can install {{site.prodname}}.
+
+1. Install the operator.
+
+   ```bash
+   kubectl create -f {{ "/manifests/tigera-operator.yaml" | absolute_url }}
+   ```
+
+1. Configure the {{site.prodname}} installation.
+
+   ```bash
+   kubectl create -f - <<EOF
+   kind: Installation
+   apiVersion: operator.tigera.io/v1
+   metadata:
+     name: default
+   spec:
+     kubernetesProvider: AKS
+     cni:
+       type: Calico
+     calicoNetwork:
+       bgp: Disabled
+       ipPools:
+        - cidr: 10.244.0.0/16
+          encapsulation: VXLAN
+   EOF
+   ```
+
+1. Confirm that all of the pods are running with the following command.
+
+   ```
+   watch kubectl get pods -n calico-system
+   ```
+
+   Wait until each pod has the `STATUS` of `Running`.
+ 
+
 ### Next steps
 
 **Required**
```

Comment:
Hmmm.  I disagree that installing `calicoctl` is a required next step, since we've installed with operator, which has automatically installed the calico apiserver.